black_hole/discord.py

```diff
@@ -138,9 +152,16 @@ async def bridge(self, room, msg, member, source):
 
         log.debug("adding message to queue")
 
+        reply_message_id = (
+            None if msg.xep0308_replace is None else msg.xep0308_replace.id_
```

Comment:
what's going on here? can it be commented?

Reply:
messages that aren't replies to other messages just have the whole attribute set to None

Reply:
ah, okay, in this case could we rename the variable to `original_message_id`? i think that conveys the intent a bit better. (from what i can understand the message we get now is an edit of an older message, so this should be the original yeah?)

Reply:
also sorry for these nits but i haven't touched this codebase in months and i like to put emphasis on high readability 😅